### PR TITLE
[torch] Add OnnxToTorch lowering for `onnx.HammingWindow`

### PR DESCRIPTION
**Pull request by vinayakdsci**
Adds OnnxToTorch lowering for the `onnx.HammingWindow` op.

### CODE REVIEW

**Comment by vinayakdsci** [comment]
cc @renxida.

File: lib/Conversion/TorchOnnxToTorch/DefaultDomainAtoF.cpp
Comment (renxida) [inline lib/Conversion/TorchOnnxToTorch/DefaultDomainAtoF.cpp:2367]: I've seen @rsuderman use a constant variable at the top of the function to cache binder.getLoc and rewriter.getF646ype. but for something as simple as this, it might be easier to do it your way. i'll leave this a tradeoff for you to decide based on readability.

**Review by renxida — APPROVED** [review]
very nice. feel free to merge after merging numerical test.

File: lib/Conversion/TorchOnnxToTorch/DefaultDomainAtoF.cpp
Comment (vinayakdsci) [reply lib/Conversion/TorchOnnxToTorch/DefaultDomainAtoF.cpp:2367]: @renxida Thanks for the approval! I think that the values are repeated only thrice, so I am not sure if it would benefit as much from caching the values, but I would be happy to change the code if you do think that it might seriously affect readability.

Also, I did write the numerical test -> https://github.com/nod-ai/SHARK-TestSuite/pull/222 .
But it requires your approval before merging. I hope you could have a look at that too.

Thanks!

Comment (vinayakdsci) [reply lib/Conversion/TorchOnnxToTorch/DefaultDomainAtoF.cpp]: No worries, I stored binder.getLoc() in a variable, and switched from getFloatAttr() to getF64FloatAttr(). I think it is much better now, thanks for the suggestion!

**Comment by vivekkhandelwal1** [comment]
@vinayakdsci, it would be helpful if you can leave a comment on https://github.com/llvm/torch-mlir/pull/3283 when you start working on an op so that no two person works on the same op.

**Comment by renxida** [comment]
https://github.com/nod-ai/SHARK-TestSuite/pull/222

Tested here

**Pull request merged**
